astronomer/airflow/version_check/update_checks.py

```diff
@@ -457,11 +462,34 @@ def available_eol(self) -> dict[str, Any] | None:
         )
         return self.get_eol_notice(current_version)
 
+    def available_yanked(self) -> dict[str, Any] | None:
+        """Check if the current version of Astronomer Runtime is yanked."""
+        from .models import AstronomerAvailableVersion
+
+        session = get_state(app=current_app).db.session
+        runtime_version = version.parse(get_runtime_version())
+        current_version = (
+            session.query(AstronomerAvailableVersion)
+            .filter(AstronomerAvailableVersion.version == str(runtime_version))
```

Comment:
Minor thing, but since this will be done on every page, it'd be better to filter on yanked here and only return a row if the version is yanked.

Reply:
Changed it